include/af/graphics.h

```diff
@@ -485,6 +485,22 @@ class AFAPI Window {
                            const char * const ytitle = "Y-Axis",
                            const char * const ztitle = NULL);
 #endif
+
+#if AF_API_VERSION >= 37
+        /**
+           Setup the axes label formats for charts
+
+           \param[in] xformat is a printf-style format specifier for x-axis
+           \param[in] yformat is a printf-style format specifier for y-axis
+           \param[in] zformat is a printf-style format specifier for z-axis
+
+           \ingroup gfx_func_window
+        */
+        void setAxesLabelFormat(const char *const xtitle = "4.1%f",
+                                const char *const ytitle = "4.1%f",
+                                const char *const ztitle = NULL);
```

Comment:
The argument names here doesn't match with the definition in src/api/cpp/graphics.cpp:192, and also those in the docstring. The CI's docs check also complains about it. I think you meant `x/y/zformat` here?